src/decision/include/subroutines/bottom_line/FindLine.h

```diff
@@ -0,0 +1,36 @@
+//
+// Created by drdc-s2632 on 17/07/18.
+//
+
+#ifndef PROJECT_FINDLINE_H
+#define PROJECT_FINDLINE_H
+
+#include "Subroutine.h"
+#include <line_detect/LineDetectMsg.h>
+
+/*** Communicating Class {findLine, adjustToLine, followLine} ***/
+class FindLine : public Subroutine {
+  public:
+    FindLine(std::unordered_map<std::string, double>* constants)
+      : Subroutine(constants) {}
+    std::string getName() override { return "FindLine"; }
+
+    std::vector<ros::Subscriber> getSubscriptions(ros::NodeHandle nh) override;
+
+  private:
+    ros::Timer timer_;
+
+    double lateralVelocityDirection_;
```

Comment:
Descriptive comment would be helpful for this? Not immediatly clear what it's for.

Reply:
fixed